src/ScottPlot5/Sandbox/ScottPlot.Sandbox.WPF/MainWindow.xaml.cs

```diff
@@ -25,6 +25,10 @@ public MainWindow()
             WpfPlot.Plot.AddScatter(Generate.Consecutive(N), Generate.Cos(N), Colors.Red);
             WpfPlot.Plot.AddScatter(Generate.Consecutive(N), Generate.Sin(N, 0.5), Colors.Green);
 
+            // swap left and right buttons
```

Comment:
Not sure if you want to undo this?

Reply:
It's in there for now to demo how to use it. I'll keep it a little longer while I keep refining this backend. Thanks for pointing it out though!